### PR TITLE
Tweak CSS of transition banner

### PR DESCRIPTION
**Pull request by benfrancis**
There's still one more tweak I'd like to do (bloody vertical aligning text...), but this is the CSS for the transition banner. I had to make one tiny tweak to the HTML, hope that's OK.

### CODE REVIEW

File: static/index.html [hunk 1]
```diff
@@ -607,13 +607,13 @@ <h3 class="logs-header" data-l10n-id="logs"></h3>
...
-           target="_blank" rel="noopener" data-l10n-id="action-required-more-info"></a>
-      </div>
-      <div id="action-required-area-buttons">
+      <div id="action-required-area-content">
+        <div id="action-required-area-text">
+          <span id="action-required-action-required" data-l10n-id="action-required"></span>
+          <span id="action-required-text" data-l10n-id="action-required-message"></span>
+          <a href="https://discourse.mozilla.org/t/an-important-update-on-mozilla-webthings/67764"
+             target="_blank" rel="noopener" data-l10n-id="action-required-more-info"></a>
+        </div>
```
Comment (benfrancis) [inline static/index.html:616]: That diff makes it look way worse than it is!

**Comment by mrstegeman** [comment]
Hmm... Things don't look quite right when the text is 1 or 3 lines.

Screenshot_2020-11-20 WebThings Gateway


Screenshot_2020-11-20 WebThings Gateway(1)


Also, if you rebase on master, the CI tests should pass.

**Comment by benfrancis** [comment]
> Hmm... Things don't look quite right when the text is 1 or 3 lines.

Yes sorry, that's the one more tweak I was talking about :) Should be fixed now.

**Comment by mrstegeman** [comment]
Looks great! Going to merge this.

**Pull request merged**